internal/pkg/describe/service.go

```diff
@@ -475,6 +475,26 @@ func (e containerEnvVars) humanString(w io.Writer) {
 	printTable(w, headers, rows)
 }
 
+// envSecret contains secrets for an rdws service.
+type envSecret struct {
+	Environment string `json:"environment"`
+	Name        string `json:"name"`
+	ValueFrom   string `json:"value"`
+}
+
+type envSecrets []*envSecret
+
+func (e envSecrets) humanString(w io.Writer) {
+	headers := []string{"Name", "Environment", "Value"}
+	var rows [][]string
+	sort.SliceStable(e, func(i, j int) bool { return e[i].Environment < e[j].Environment })
+	sort.SliceStable(e, func(i, j int) bool { return e[i].Name < e[j].Name })
```

Comment:
nit: perhaps calling `SliceStable` twice gives us the result that we want for now, but it might not in the future because go lib is free to change whatever algorithm they use under the hood. I think this ⬇️ should guarantee to get what's intended here
```suggestion
	sort.SliceStable(e, func(i, j int) bool { 
	  if e[i].Name == e[j].Name {
	     return  e[i].Environment < e[j].Environment
	  }
	return e[i].Name < e[j].Name 
	})
```

Reply:
Updated in next commit.